js/main.js

```diff
@@ -71,17 +73,39 @@ function applySpeed(value, ignoreDeath = false) {
     return value * getGameSpeed(ignoreDeath) / updateSpeed;
 }
 
-function getPopulation() {
-    let population = 0;
+//Need to review formula + application in updatePopulation()
+function updateHeat() {
+    let danger = getEffectFromOperations(EffectType.Danger);
+    danger += gameData.currentSkill.getEffect(EffectType.Danger);
+    const military = getEffectFromOperations(EffectType.Military);
+
+    if (danger === military) {
+        return;
+    }
+
+    gameData.heat = Math.sign(danger-military) * Math.log10(Math.abs(danger - military));
+}
+
+function updatePopulation(){
+    const growth = getEffectFromOperations(EffectType.Growth);
+    gameData.population += applySpeed(growth - (0.01 * gameData.population) * (1 - Math.log10(1 + gameData.heat)));
+    gameData.population = Math.max(gameData.population, 0);

```

Comment:
min population ist 1 weil Captain und weil das Ding imho iwo als log oder factor verwendet werden soll 🙃 (ich weiß, is wieder nirgends dokumentiert)